docs/components/Combobox/Combobox.stories.mdx

```diff
@@ -50,14 +53,25 @@ activator.
 ##### Custom activator
 
 The custom activator should be used in any instance where the default activator
-is not suitable. Your custom activator should be a child of
-`<Combobox.Activator />`. An example of using a custom activator can be found in
-the
+is not suitable.
+
+**Child React Component:** To use a customized `Chip` or `Button` your custom
+activator should be a child of `<Combobox.Activator />`. An example of using a
+custom activator can be found in the
 [custom activator](../?path=/story/components-selections-combobox-web--custom-activator)
 story.
 
-Omitting a custom activator subcomponent will result in the default activator
-being used.
+**Child Render Function:** If neither a Chip nor a Button is sufficient for your
+needs, you may instead provide a function as the child to `Combobox.Activator`.
+
+As arguments, the function will receive the toggle method to swap between open
+and closed states, as well as accessibility attributes required for a Combobox.
+_You must implement these yourself_. See the custom example for reference.
```

Comment:
```suggestion
**You must implement these yourself**. See the custom example for reference.
```
I think bolding this might make it "pop" out more

Reply:
Just confirming the markdown I gave was valid

Reply:
The markdown is valid if we want to change it
<img width="818" alt="image" src="https://github.com/user-attachments/assets/18a33b25-8d1d-44d8-93e4-6035b9e3e843">

Reply:
The markdown I gave is valid

Reply:
yeah I think I like that better. if we're going ahead with the DIY-approach it's pretty important to call out.

I started running out of heading levels and had to use bold as the `Child Render Function` bit to make it stand out which is probably a signal this document is getting a bit big.